frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/projects/ProjectPortlet.java

```diff
@@ -239,6 +239,13 @@ private void saveAttachmentUsages(ResourceRequest request, ResourceResponse resp
 
     }
 
+    private void setProjectPathToAttachmentUsages(List<AttachmentUsage> usagesToCreate, Project project) {
+        usagesToCreate.forEach(usage -> {
+            if (!usage.usageData.getLicenseInfo().isSetProjectPath())
```

Comment:
Should we maybe check that `usage.usageData.getLicenseInfo() != null` ?

Reply:
@nutanv1contr I am not sure if that is relevant, but could you check if that makes sense? If it was incorporated, please comment to the question to make the situation clear.

Reply:
Sorry I missed add my comment . I have modified and refactored the changes as per review comment.